build.gradle.kts

```diff
@@ -47,7 +47,7 @@ dependencies {
     testCompile("org.testfx:testfx-junit:4.0.13-alpha")
 
     testCompile("org.mockito:mockito-core:1.10.19")
-    testCompile("com.nhaarman.mockito-kotlin:2.1.0") {
```

Comment:
@ahinchman1 Looks like this namespace is potentially incorrect? Updated it, not sure if I got the right one.